cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -2122,6 +2122,8 @@ spec:
                         type: object
                       nullable: true
                       type: array
+                    skipApplyAllPlacement:
```

Comment:
New field should be updated in `ceph-cluster-crd.md` file.

Reply:
added